corehq/apps/geospatial/reports.py

```diff
@@ -42,22 +44,48 @@ def template_context(self):
     @property
     def report_context(self):
         cases = []
+        invalid_geo_cases_count = 0
         for row in self.es_results['hits'].get('hits', []):
             es_case = self.get_case(row)
             display = CaseDisplayES(es_case, self.timezone, self.individual)
+
+            coordinates = _get_geo_location(es_case)
+            if coordinates is None:
+                invalid_geo_cases_count += 1
+                continue
             # We should consider passing in a "center_coordinates" fields to center the map
             # to the relavent
             case = {
                 "case_id": display.case_id,
                 "case_type": display.case_type,
                 "name": display.case_name,
-                "coordinates": _get_geo_location(es_case)
+                "coordinates": coordinates
             }
             cases.append(case)
+
+        invalid_cases_link = self._invalid_geo_cases_report_link if invalid_geo_cases_count else ''
+
         return dict(
-            context={"cases": cases},
+            context={
+                "cases": cases,
+                "invalid_geo_cases_report_link": invalid_cases_link,
+            },
         )
 
     @property
     def default_report_url(self):
         return reverse('geospatial_default', args=[self.request.project.name])
+
+    @property
+    def _invalid_geo_cases_report_link(self):
+        query = self.request.GET.copy()
```

Comment:
Use all the defined filters such that it's predefined on the CLE page.

Reply:
Can we add a comment in the code saying this?